Example/Custom Integration/BrowseExamplesViewController.h

```diff
@@ -30,8 +30,6 @@ typedef void (^STPConfirmPaymentIntentCompletionHandler)(STPBackendResult status
                                   paymentMethod:(NSString *)paymentMethodID
                                       returnURL:(NSString *)returnURL
                                      completion:(STPPaymentIntentCreateAndConfirmHandler)completion;
-- (void)confirmPaymentIntent:(STPPaymentIntent *)paymentIntent completion:(STPConfirmPaymentIntentCompletionHandler)completion;
```

Comment:
`CardManualViewController` should be using this but currently doesn't, we should keep this method.  We don't need to return a `STPPaymentIntent` in the completion handler, it could just be `(STPBackendResult status, nullable NSError *error)`.

I'm fixing this in my last SetupIntents PR.